packages/frontend/index.html

```diff
@@ -5,36 +5,14 @@
         <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.css">
         <script src="//ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
         <script src="//cdn.jsdelivr.net/npm/semantic-ui@2.4.2/dist/semantic.min.js"></script>
-        <style>
-            #container {
-                padding-bottom: 50px;
-                padding-top: 50px;
-            }
-            #socket-input { width: 450px; }
-            #socket-send { margin-right: 20px; }
-            #socket-broadcast { margin-right: 20px; }
-            .c6o-panel { background-color: lightgrey; }
-            .logo {
-                height: 40px;
-                margin-right: 12px;
-                width: 40px;
-            }
-            pre {
-                white-space: pre-wrap;       /* Since CSS 2.1 */
-                white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
-                white-space: -pre-wrap;      /* Opera 4-6 */
-                white-space: -o-pre-wrap;    /* Opera 7 */
-                word-wrap: break-word;       /* Internet Explorer 5.5+ */
-            }
-            section { margin-bottom: 30px; }
-        </style>
+        <link rel="stylesheet" href="/index.css">
     </head>
     <body>
         <div id="container" class="ui centered aligned container">
             <div class="ui grid">
                 <div class="two column row">
                     <div class="left floated middle aligned column">
-                        <h1 class='border-1'><img src='c6o.png' class='logo'>Sample Kubernetes Project</h1>
+                        <h1 class='border-1'><img src='c6o.png' class='logo'>CodeZero Sample Project</h1>
```

Comment:
Need to be consistent with the docs. Also, don't have to mention the company name everywhere. We already have the logo.
```suggestion
                        <h1 class='border-1'><img src='c6o.png' class='logo'>Sample Project for Kubernetes</h1>
```